toolkit/tools/imager/imager.go

```diff
@@ -612,37 +612,37 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap,
 			err = fmt.Errorf("failed to configure boot loader: %w", err)
 			return
 		}
+	}
 
-		// Preconfigure SELinux labels now since all the changes to the filesystem should be done
-		if systemConfig.KernelCommandLine.SELinux != configuration.SELinuxOff {
-			err = installutils.SELinuxConfigure(systemConfig, installChroot, mountPointToFsTypeMap)
-			if err != nil {
-				err = fmt.Errorf("failed to configure selinux: %w", err)
-				return
-			}
+	// Preconfigure SELinux labels now since all the changes to the filesystem should be done
+	if systemConfig.KernelCommandLine.SELinux != configuration.SELinuxOff {
```

Comment:
Should you be re-adding the `!isRootFS` check over the `ReadOnlyVerityRoot` stuff?

Reply:
what do you mean?

Reply:
Previously, the `if !isRootFS` check covered the bootloader, SELinux, and `ReadOnlyVerityRoot` stuff. This change removed the `if !isRootFS` check from both the SELinux and `ReadOnlyVerityRoot` stuff. Did you intend to remove it from the `ReadOnlyVerityRoot` stuff?

Reply:
That is `if systemConfig.ReadOnlyVerityRoot.Enable {` on line 627.